### PR TITLE
Emitter stacking fix

### LINKED ISSUE #14109: Emmitters can be screwdriverd and crowbarred when locked / on
**Issue Description**:

Emitters can be screwdrivered / crowbarred when locked / on, when it is supposed to be tamper proof

**What did you expect to happen**: 

For a message to display that you can not do this when it is locked / firing. 

**What happened instead**:

You can open  it, breaking the sprite, and then crowbar it, deconstructing it when it is supposed to be "relatively" tamper proof

**Why is this bad/What are the consequences**:

Broken sprites, easy to deconstruct then should be.
https://imgur.com/a/rQYaTcY 
**Steps to reproduce the problem**:

Setup an emitter
Lock it.
Screwdriver it.
Crowbar it.

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Refactors emitters a bit, adds a check for if there's already an emitter on the tile, and adds directional sprites for emitters with their panel open.
Fixes #14109

## Why It's Good For The Game

See below.

## Images of changes

**Fixes this:**
hello coding_chat!

## Changelog
:cl:
del: Removed the ability to use a multitool on an emitter, since it didn't actually do anything.
tweak: Anchored emitters now have a limit of 1 per tile.
imageadd: Added directional sprites for emitters with their maintenance panel open.
/:cl:
*(Probably missing something here but I'm tired and someone else will notice if I don't. :P)*

### CODE REVIEW

**Comment by Sneedstar** [comment]
I will find a way around this.

File: code/modules/power/singularity/emitter.dm [hunk 1]
```diff
@@ -87,77 +104,185 @@
...
+		anchored = FALSE
+		user.visible_message("[user] unsecures [src] reinforcing bolts from the floor.",
+			"You undo the external reinforcing bolts.",
+			"You hear a ratchet")
+	playsound(src, I.usesound, I.tool_volume, TRUE)
+
+/obj/machinery/power/emitter/screwdriver_act(mob/living/user, obj/item/I) // Multitool menu NYI
+	. = TRUE
+	if(state != EMITTER_WELDED)
+		return
```
Comment (SteelSlayer) [inline code/modules/power/singularity/emitter.dm:208]: Before you could screwdriver emitters open at any time, but now it has to be welded. Is this an intended change?

Comment (SabreML) [reply code/modules/power/singularity/emitter.dm:208]: I'm pretty sure there *was* a good reason for this, but if there was I've forgotten it now.
All seems to work fine with that removed anyway.

**Review by SteelSlayer — APPROVED** [review]

**Comment by Fox-McCloud** [comment]
if you're going to  re-style everything, mind leaving a comment where the actual changes are?

**Comment by SabreML** [comment]
> if you're going to re-style everything, mind leaving a comment where the actual changes are?

I put it into a separate commit for exactly that reason.
https://github.com/ParadiseSS13/Paradise/pull/15379/commits/ba035803642cf9685a88b6b8f39c895d8a7eb145

File: code/modules/power/singularity/emitter.dm [hunk 2]
```diff
@@ -87,77 +104,170 @@
...
+	. = TRUE
+	if(active)
+		to_chat(user, "<span class='warning'>Turn off [src] first!</span>")
+		return
+	if(state == EMITTER_WELDED)
+		to_chat(user, "<span class='warning'>[src] needs to be unwelded from the floor!</span>")
+		return
+
+	if(state == EMITTER_NEEDS_WRENCH)
+		for(var/obj/machinery/power/emitter/E in loc)
```
Comment (AffectedArc07) [inline code/modules/power/singularity/emitter.dm:188]:
```suggestion
		for(var/obj/machinery/power/emitter/E in get_turf(src))
```
`loc` isnt always a turf

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/modules/power/singularity/emitter.dm (hunk 2 above)
Comment (SabreML) [reply code/modules/power/singularity/emitter.dm:188]: Completely missed this, whoops!

**Review by AffectedArc07 — APPROVED** [review]
In the future please put exploit fixes and entire refactors in different PRs.

Forcing us to review entire files for stuff which should be handled quickly isn't great.

**Pull request merged**